functional/security/Crypto/build.xml

```diff
@@ -51,81 +51,14 @@
 	<target name="init">
 		<mkdir dir="${DEST}" />
 	</target>
-	<if>
-		<matches pattern="^([89]|1[02-6])$" string="${JDK_VERSION}"/>
-		<then>
-			<property name="jtregTar" value="jtreg_5_1_b01"/>
-		</then>
-		<elseif>
-			<matches pattern="^11$" string="${JDK_VERSION}"/>
-			<then>
-				<property name="jtregTar" value="jtreg_6_1"/>
-			</then>
-		</elseif>
-		<elseif>
-			<!-- versions 17, 21, 22 -->
-			<matches pattern="^(17|2[12])$" string="${JDK_VERSION}"/>
-			<then>
-				<property name="jtregTar" value="jtreg_7_3_1_1"/>
-			</then>
-		</elseif>
-		<else>
-			<fail message="Unsupported JDK version: ${JDK_VERSION}"/>
-		</else>
-	</if>
-	<if>
-		<or>
-			<equals arg1="${JDK_IMPL}" arg2="ibm"  />
-			<equals arg1="${JDK_IMPL}" arg2="openj9" />
-		</or>
-		<then>
-			<property name="openj9jtregtimeouthandler" value=",tohandler_simple"/>
-		</then>
-		<else>
-			<property name="openj9jtregtimeouthandler" value=""/>
-		</else>
-	</if>
-	<property name="LIB" value="${jtregTar}${openj9jtregtimeouthandler}"/>
 	
 	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
-	<target name="getJtreg">
-		<mkdir dir="${DEST}"/>
-		<if>
-			<not>
-				<available file="${LIB_DIR}/${jtregTar}.tar.gz" />
-			</not>
-			<then>
-				<if> 
-					<available file="custom_jtreg.tar.gz"/>
-					<then>
-						<echo message="Using custom_jtreg.tar.gz"/>
-						<copy file="custom_jtreg.tar.gz" tofile="${jtregTar}.tar.gz"/>
-					</then>
-				</if>
-			</then>
-			<else>
-   				<copy file="${LIB_DIR}/${jtregTar}.tar.gz" tofile="${jtregTar}.tar.gz"/>
-			</else>
-		</if>
-		<exec executable="gzip" failonerror="true">
-			<arg line="-df ${jtregTar}.tar.gz" />
-		</exec>
-		<if>
-			<contains string="${SPEC}" substring="zos" />
-			<then>
-				<exec executable="tar" failonerror="true">
-					<arg line="xfo ${jtregTar}.tar -C ${DEST}" />
-				</exec>
-			</then>
-			<else>
-				<exec executable="sh" failonerror="true">
-					<arg line="-c 'cat ${jtregTar}.tar | (cd ${DEST} &amp;&amp; tar xof -)'" />
-				</exec>
-			</else>
-		</if>
-	</target>
+
+	<!-- Call getJtregVersion, getJtreg target from getDependencies.xml -->
+    <target name="invokeGetJtreg" depends="getJtregVersion,getJtreg">
+    </target>
 	
-	<target name="dist" depends="getDependentLibs,getJtreg,getCryptoTest" description="generate the distribution">
+	<target name="dist" depends="invokeGetJtreg,getCryptoTest" description="generate the distribution">
```

Comment:
Just call `getJtreg` instead and remove line 57 - 59. In `getJtreg` (TKG repo), it depends on `getJtregVersion`.